packages/fxa-settings/src/components/DataBlock/index.tsx

```diff
@@ -56,10 +57,11 @@ export const DataBlock = ({
   return (
     <div className="flex flex-col items-center">
       <div
-        className={`flex rounded-xl px-7 font-mono text-center text-sm font-bold text-black bg-gradient-to-tr from-blue-600/10 to-purple-500/10 flex-wrap relative mb-6 ${
-          valueIsArray ? 'max-w-sm py-4' : 'max-w-lg py-5'
-        } ${isInline ? 'gap-6 items-center' : ''}
-        }`}
+        className={classNames(
+          'relative flex rounded-lg px-6 font-mono text-center text-sm font-bold text-black bg-gradient-to-tr from-blue-600/10 to-purple-500/10 mb-2',
```

Comment:
This still looks taller and more round to me than Figma but I'm not sure if that's totally up to date and I think this looks nice anyways.

Reply:
It's a tad bigger because I made sure the tap area of the copy icon was at least 44px  for a11y 🙂